### PR TITLE
[Backport][V1.3] [MME] fix incorrect formatting of time/timezone in EMM-information pa…

### PR DESCRIPTION
**Pull request by ssanadhya**
Back-porting #3998 to branch v1.3 after resolving merge conflicts

2 Major issues:

* timezone has to be computed using 2 characters
* time has to follow 3GPP syntax

Fix a few compilation warnings.

Signed-off-by: Raphael Defosseux <raphael.defosseux@openairinterface.org>
Signed-off-by: Shruti Sanadhya <ssanadhya@fb.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by ulaskozat — APPROVED** [review]
looks good to go.

**Review by rdefosse — APPROVED** [review]
Shall I approve my own PR? yes!

File: lte/gateway/docker/mme/Dockerfile [hunk 1]
```diff
@@ -283,16 +285,16 @@ ENV C_BUILD=/build/c
...
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get upgrade --yes && DEBIAN_FRONTEND=noninteractive apt-get install --yes \
-    psmisc \
-    openssl \
-    net-tools \
-    tshark \
-    redis-server \
+ENV TZ=Europe/Paris
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get upgrade --yes && \
```
Comment (tmdzk) [inline lte/gateway/docker/mme/Dockerfile:291]: The `DEBIAN_FRONTEND=noninteractive` is already set above you don't need it on the next apt installs

**Review by tmdzk — APPROVED** [review]
LGTM

File: lte/gateway/docker/mme/Dockerfile (hunk 1 above)
Comment (ssanadhya) [reply lte/gateway/docker/mme/Dockerfile:291]: Will fix forward in master branch.

**Pull request merged**